[2023-05-04T10:30:11.682+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chatgpt_dag.chatgpt_saveresponse scheduled__2023-05-04T10:25:00+00:00 [queued]>
[2023-05-04T10:30:11.687+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chatgpt_dag.chatgpt_saveresponse scheduled__2023-05-04T10:25:00+00:00 [queued]>
[2023-05-04T10:30:11.687+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-05-04T10:30:11.687+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-05-04T10:30:11.687+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-05-04T10:30:11.696+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): chatgpt_saveresponse> on 2023-05-04 10:25:00+00:00
[2023-05-04T10:30:11.699+0000] {standard_task_runner.py:55} INFO - Started process 291166 to run task
[2023-05-04T10:30:11.701+0000] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'chatgpt_dag', 'chatgpt_saveresponse', 'scheduled__2023-05-04T10:25:00+00:00', '--job-id', '3489', '--raw', '--subdir', 'DAGS_FOLDER/chatgpt_dag.py', '--cfg-path', '/tmp/tmpvzkj8tvp']
[2023-05-04T10:30:11.702+0000] {standard_task_runner.py:83} INFO - Job 3489: Subtask chatgpt_saveresponse
[2023-05-04T10:30:11.730+0000] {task_command.py:389} INFO - Running <TaskInstance: chatgpt_dag.chatgpt_saveresponse scheduled__2023-05-04T10:25:00+00:00 [running]> on host trivia-machine.us-central1-a.c.triviapractice-381517.internal
[2023-05-04T10:30:11.772+0000] {taskinstance.py:1516} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=richc
AIRFLOW_CTX_DAG_ID=chatgpt_dag
AIRFLOW_CTX_TASK_ID=chatgpt_saveresponse
AIRFLOW_CTX_EXECUTION_DATE=2023-05-04T10:25:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-05-04T10:25:00+00:00
[2023-05-04T10:30:11.788+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-05-04T10:30:11.790+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=chatgpt_dag, task_id=chatgpt_saveresponse, execution_date=20230504T102500, start_date=20230504T103011, end_date=20230504T103011
[2023-05-04T10:30:11.833+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-05-04T10:30:11.845+0000] {taskinstance.py:2596} INFO - 1 downstream tasks scheduled from follow-on schedule check
